charts/team-ns/templates/argocd/argocd-applicationset.yaml

```diff
@@ -65,6 +71,10 @@ spec:
         {{- if eq $v.teamId "admin" }} 
         syncOptions:
           - RespectIgnoreDifferences=true
+          - ServerSideApply=true
+          {{- if .createNamespace }}
+          - CreateNamespace=true
```

Comment:
1) What happens to the namespace once the workload (ApplicationSet) is removed. 
2) Does recreating a workload into the same namespace cause any issue?
3) What if two workloads are defined and target the same namespace (both with create namespace)? Should it be allowed?

Reply:
we see that NS is not removed once workload is removed.

Reply:
1. no
2. no
3. no